distributed/http/worker/prometheus/core.py

```diff
@@ -20,52 +22,54 @@ def collect(self):
         from prometheus_client.core import GaugeMetricFamily
 
         tasks = GaugeMetricFamily(
-            "dask_worker_tasks", "Number of tasks at worker.", labels=["state"]
+            self.build_name("tasks"),
+            "Number of tasks at worker.",
+            labels=["state"],
         )
-        tasks.add_metric(["stored"], len(self.worker.data))
-        tasks.add_metric(["executing"], self.worker.executing_count)
-        tasks.add_metric(["ready"], len(self.worker.ready))
-        tasks.add_metric(["waiting"], self.worker.waiting_for_data_count)
-        tasks.add_metric(["serving"], len(self.worker._comms))
+        tasks.add_metric(["stored"], len(self.server.data))
+        tasks.add_metric(["executing"], self.server.executing_count)
+        tasks.add_metric(["ready"], len(self.server.ready))
+        tasks.add_metric(["waiting"], self.server.waiting_for_data_count)
+        tasks.add_metric(["serving"], len(self.server._comms))
         yield tasks
 
         yield GaugeMetricFamily(
-            "dask_worker_connections",
+            self.build_name("connections"),
```

Comment:
Similar here. Technically this is not the number of connections but the number of concurrent fetch requests. There are many more connections out there